.github/dependabot.yml

```
@@ -27,3 +27,25 @@ updates:
       - "matomo-org/core-reviewers"
     pull-request-branch-name:
       separator: "-" 
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+    labels:
+      - "dependencies"
+    groups:
+      all-dependencies:
+        patterns:
+          - "*"
+    commit-message:
+      prefix: "[NPM] "
+    reviewers:
+      - "sgiehl"
```

Should this be the core reviewers?

---

sure. missed to update that when copying that over 🙈 
```suggestion
      - "matomo-org/core-reviewers"
```